rockcraft/services/package.py

```diff
@@ -0,0 +1,66 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Rockcraft Package service."""
+
+from __future__ import annotations
+
+import pathlib
+
+from craft_application import AppMetadata, models, services
+from overrides import override
+
+from rockcraft.models.project import Project
+
+
+class RockcraftPackageService(services.PackageService):
```

Comment:
This one is a "nop" for now; needed because the package service is mandatory and has no default

Reply:
With the change to give each service access to communicate with the other services, are there any blockers remaining to move the business logic from `rockcraft.lifecycle._pack` here?

(Not appropriate for this PR, just wondering whether you immediately know of anything that would need adjustment)

Reply:
nope! yay